### ISSUE TITLE
Add billable toggle to activity

### ISSUE THREAD

**Issue by curlyjason** [OrigamiStructures/foldingTime#13]
Need to reveal on add and edit forms. May have this influence background color. Need to be able to sort in the reporting module, too.

**Comment by dreamingmind** [comment]
Yes, there were two new status fields we were adding to those records I think

**Comment by curlyjason** [comment]
Crap! We had a bunch of discussion about that, but I don't remember a thing about what we were adding there. It had to do with billable and billed, right?

**Comment by dreamingmind** [comment]
and invoiced, reported, etc

**Comment by curlyjason** [comment]
Right. I'm wondering if those are stages like 'active, inactive'
So it would be like 'active, reported, invoiced, complete'
Maybe there's a 'killed' stage as well?

**Comment by dreamingmind** [comment]
they were state-like as I remember. though not strictly sequential.

and there were two because of the 'contractor' situation, or that was the thinking at the time. It would allow a record to be marked as reported to you but not invoiced... hmm... don't really remember.

**Comment by curlyjason** [comment]
I see the non necessarily sequential.
Using them in the same way, though, would help keep everything straight. I think either one of us could mark time as active, complete, reported and invoiced. The active and complete is handled as the regular time duties (we've got paused in there as well). The reported and invoiced would be triggered through a group process.
1. Migrate the sorting routines into a specialized setup for doing time reports.
2. Setup a process to allow the creation of a time report.
3. Items added to the report will be marked as reported. The report will get it's own ID.
4. bringing up a report ID would allow me to mark that report as invoiced, enter the invoice number, and move the entire set to complete.
5. There would be an option to kill individual time records.

**Comment by dreamingmind** [comment]
that's starting to flesh out a bit.
